Amplify/Core/Support/TimeInterval+Helper.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+extension TimeInterval {
+
+    public static func seconds(_ value: Double) -> TimeInterval {
```

Comment:
The return value is in seconds (because `TimeInterval` is a Double that is used as a seconds value), but the arguments are in the units specified by the function name. In any case, the caller can't specify the argument at the call site, but rather passes the value bare to the method. Thus `TimeInterval.seconds(1)` returns 1 second; `TimeInterval.minutes(30)` returns 30*60=1800 seconds, and so on. 

This is an equivalent usage to the Foundation `DispatchTime` class, which offers convenience methods like `DispatchTime.minutes()`, etc, and enables constructs like `let t = DispatchTime(.now() + .seconds(30))`.

It's possible I'm misunderstanding your point :)

Reply:
Ah! I got it.

This says "Give me a time interval with this many seconds." As opposed to, "Convert this quantity of time (in what units?) _into_ seconds."

My brain was stuck in the Java `TimeUnit` utility:
```
// Converts 5,000 milliseconds (the provided value) into units of seconds. Returns 5.
TimeUnit.MILLISECONDS.toSeconds(5_000);
```

Reply:
i am optimistically resolving this.